pallets/author-slot-filter/src/lib.rs

```diff
@@ -157,6 +157,13 @@ pub mod pallet {
 	/// The type of eligibility to use
 	pub type EligibilityValue = NonZeroU32;
 
+	impl EligibilityValue {
+		/// Default total number of eligible authors, must NOT be 0.
+		pub fn default() -> Self {
+			NonZeroU32::new_unchecked(50)
```

Comment:
In what cases is this value expected to be relevant? If it's mostly supposed to be a safe number, I'd set it to 100%.